pkg/kubectl/cmd/get.go

```diff
@@ -316,12 +316,9 @@ func RunGet(f *cmdutil.Factory, out io.Writer, errOut io.Writer, cmd *cobra.Comm
 			return err
 		}
 
-		for ix := range infos {
-			objs[ix], err = infos[ix].Mapping.ConvertToVersion(infos[ix].Object, version)
-			if err != nil {
-				allErrs = append(allErrs, err)
-				continue
-			}
+		objs, err = resource.AsVersionedObjects(infos, version, f.JSONEncoder())
```

Comment:
@deads2k for the old convert: `infos[ix].Mapping.ConvertToVersion(infos[ix].Object, version)`, since the version is always `v1`, this conversion will fail, and cause this panic. For `resource.AsVersionedObjects` it will respect object's mapping version.